.github/workflows/static-analyze.yml

```diff
@@ -19,7 +19,8 @@ jobs:
         dependencies:
           - "locked"
         php-version:
-          - "7.4"
+          - "8.1"
+          - "8.2"
```

Comment:
might be worth adding also 8.3, should work out of the box

Reply:
done